### ISSUE TITLE
Hi, this happened on one of our production customer devices.

### ISSUE THREAD

**Issue by wegileShashankRawat** [Instabug/Instabug-Android#301]
### Steps to Reproduce the Problem
   Don't know, as I am getting the crash on my crash analytics reports.

### Expected Behavior
   No more crashes .

### Actual Behavior
 Crash with this stacktrace:

Fatal Exception: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String com.instabug.library.internal.storage.cache.Cache.getId()' on a null object reference
       at com.instabug.survey.cache.SurveysCacheManager.migrateOldCacheToDb(SurveysCacheManager.java:16)
       at com.instabug.survey.SurveyPlugin$d.run(SurveyPlugin.java:9)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
       at com.instabug.library.util.threading.b$a.run(PriorityThreadFactory.java:8)
       at java.lang.Thread.run(Thread.java:764)

### Instabug integration code
new Instabug.Builder(this, getResources().getString(R.string.instabug_app_token))
                    .setInvocationEvents(isShake ? InstabugInvocationEvent.SHAKE : InstabugInvocationEvent.NONE)
                    .setInstabugLogState(Feature.State.DISABLED) // Disbale crash reporting, as we have Google Analytics.
                    .build();

            BugReporting.setReportTypes(BugReporting.ReportType.BUG, BugReporting.ReportType.FEEDBACK);

            Instabug.enable();

### SDK Version
  'com.instabug.library:instabug:9.0.5'

### Android Version
 Android  9

### Device Model
 Google 
Chromebook Plus (V2)

**Comment by Muhammadyoussef** [comment]
Hi @wegileShashankRawat,

This issue got resolved since `v8.2.0`. Please make sure the device receiving the crash is not running an older Instabug SDK version 🙏 

Also, FYI to disabled Instabug crash reporter you need to use the following API:
```
CrashReporting.setState(DISABLED) // after initializing Instabug
```

**Comment by amrbrkt** [comment]
Hello @wegileShashankRawat 
I highly recommend updating to our latest release `10.2.0`, the crash should be fixed. 
I'm closing this issue, don't hesitate to reopen if you still have the issue.
